js/entry-points/tracing.js

```diff
@@ -140,6 +140,14 @@ audion.entryPoints.enumAudioBufferProperties_ =
 
 
 /**
+ * If set to true, Web Audio Inspector no longer tracks web audio calls so that
+ * AudioNodes can be GC-ed. This is desired when the user is not using developer
+ * tools.
+ */
+audion.entryPoints.audioUpdatesAreMissing_ = false;
```

Comment:
Unfortunately, this fix only applies if the user is not using dev tools ... as we know, we'll need some native browser signal for AudioNode GCs for an ideal solution.